app/javascript/components/components/__tests__/Header.test.js

```diff
@@ -11,4 +11,10 @@ describe("When header renders", () => {
         const renderHeaderTag = header.find("header")
         expect(renderHeaderTag.length).toEqual(1)
     })
-})
+}) 
+
+//  Aaron said we can use spec testing here
```

Comment:
What is going on here?

Reply:
Aaron was giving us an idea for a test but had to leave shortly after, so we added that as a note to come back to. We have listed that test as its own card on Trello to come back to later.

Reply:
Gotcha!